From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Thu, 6 Feb 2014 13:35:16 +0000
Subject: QJN not a macro

change line break to space to avoid this and consequent lost text

manpage-has-errors-from-man
usr/share/man/man1/qhull.1.gz 750: warning: macro `QJn'' not defined
---
 html/qhull.man | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/html/qhull.man b/html/qhull.man
index 4ede521..733982e 100644
--- a/html/qhull.man
+++ b/html/qhull.man
@@ -746,8 +746,7 @@ not increase n beyond a certain value, and it stops after a certain number
 of attempts [see user.h].  Option 'QJ'
 selects a default value for n.  The output will be simplicial.  For
 Delaunay triangulations, 'QJn' sets 'Qbb' to scale the last coordinate
-(not if 'Qbk:n' or 'QBk:n' is set).
-'QJn' is deprecated for Voronoi diagrams.  See also 'Qt'.
+(not if 'Qbk:n' or 'QBk:n' is set).  'QJn' is deprecated for Voronoi diagrams.  See also 'Qt'.
 .TP
 Qm
 Only process points that would otherwise increase max_outside.  Other
